lib/HttpClient.ts

```diff
@@ -7,6 +7,7 @@ import https = require("https");
 import tunnel = require("tunnel");
 import ifm = require('./Interfaces');
 import fs = require('fs');
+import zlib = require('zlib');
```

Comment:
The [GitHub Page](https://github.com/kkaefer/DEPRECATED-node-zlib) says this lib is deprecated. We should find another option :)